production/helm/loki/templates/backend/statefulset-backend.yaml

```diff
@@ -82,6 +82,8 @@ spec:
           volumeMounts:
             - name: config
               mountPath: /etc/loki/config
+            - name: runtime-config
```

Comment:
Same comments as for the `statefulset-write.yaml` file

Reply:
done :)